scripts/screeps_rl_training_runner.py

```
@@ -1001,6 +1009,87 @@ def execute_simulator_runs(
+    if row is None:
+        raise RuntimeError("pre-scale private-simulator trainability smoke gate produced no variant result")
+    if row.get("ok") is not True:
+        reason = text_or_none(row.get("error")) or "variant smoke result was not ok"
+        raise RuntimeError(f"pre-scale private-simulator trainability smoke gate failed: {reason}")
+    active_code_error = simulator_harness.private_simulator_active_code_readback_error(row.get("activeCodeReadback"))
+    if active_code_error is not None:
+        raise RuntimeError(f"pre-scale private-simulator trainability smoke gate failed: {active_code_error}")
+    injection = row.get("runtimeParameterInjection")
+    consumption = row.get("runtimeParameterConsumption")
+    if not isinstance(injection, dict) or injection.get("runtimeParameterInjection") is not True:
+        raise RuntimeError("pre-scale private-simulator trainability smoke gate did not inject runtime parameters")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)